<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="300" started-at="2024-04-05T16:44:35Z" finished-at="2024-04-05T16:44:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="300" started-at="2024-04-05T16:44:35Z" finished-at="2024-04-05T16:44:35Z">
      <class name="ustud">
        <test-method status="PASS" signature="f1()[pri:0, instance:ustud@1fe40d5]" name="f1" is-config="true" duration-ms="20" started-at="2024-04-05T16:44:35Z" finished-at="2024-04-05T16:44:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f1 -->
        <test-method status="FAIL" signature="testImportexport1()[pri:0, instance:ustud@1fe40d5]" name="testImportexport1" duration-ms="232" started-at="2024-04-05T16:44:35Z" finished-at="2024-04-05T16:44:35Z">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[-1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: -1
at jxl.write.biff.WritableSheetImpl.getRowRecord(WritableSheetImpl.java:1228)
at jxl.write.biff.WritableSheetImpl.addCell(WritableSheetImpl.java:1151)
at ustud.testImportexport1(ustud.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testImportexport1 -->
      </class> <!-- ustud -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
